closure/repositories.bzl

```diff
@@ -888,10 +888,9 @@ def org_jsoup():
         name = "org_jsoup",
         licenses = ["notice"],  # The MIT License
         jar_urls = [
-            "https://mirror.bazel.build/repo1.maven.org/maven2/org/jsoup/jsoup/1.10.2/jsoup-1.10.2.jar",
-            "https://repo1.maven.org/maven2/org/jsoup/jsoup/1.10.2/jsoup-1.10.2.jar",
+            "https://repo1.maven.org/maven2/org/jsoup/jsoup/1.11.3/jsoup-1.11.3.jar",
```

Comment:
@dslomov rules_closure generally provides at least 2 URLs including bazel.build ones. What's the recommended policy here?

Reply:
I am not sure how to upload into mirror.bazel.build

Reply:
Friendly ping